css/styles.css

```diff
@@ -73,7 +62,94 @@
 }
 
 .body {
-  font-family: sans-serif;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: #172b4d;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.working {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.card {
+  width: 270px;
+  margin: 10px;
+  border: solid;
+  border-color: #d0d9d4;
+  border-radius: 15px;
+}
+
+.card-image {
+  background-color: #d0d9d4;
+  height: 270px;
+  margin-bottom: 15px;
+  background-size: cover;
+  border-radius: 15px 15px 0 0;
+}
+
+.card-text {
+  font-family: 'Inter', sans-serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #3a4a42;
+  padding-left: 0;
+}
+
+.alist {
```

Comment:
- [ ] Kindly apply `padding-left: 0;` to this element as suggested by the previous reviewer 👍